.github/workflows/release.yaml

```diff
@@ -0,0 +1,57 @@
+name: Release
+on:
+  push:
+    tags:
+      - "v*"
+env:
+  GH_USER: aqua-bot
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Get the semantic version
+        run: |
+          echo "RELEASE_VERSION=${GITHUB_REF/refs\/tags\/v/}" >> $GITHUB_ENV
+      - name: Get the minor version
+        run: |
+          echo "MINOR_VERSION=$(echo ${RELEASE_VERSION} | cut -d. -f1,2)" >> $GITHUB_ENV
+      - name: Get the major version
+        run: |
+          echo "MAJOR_VERSION=$(echo ${RELEASE_VERSION} | cut -d. -f1)" >> $GITHUB_ENV
+      - name: Create directories
+        run: mkdir -p bundle/docker bundle/kubernetes 
+      - name: Copy manifest
+        run: |
+          cp .manifest bundle/
+          sed -i -e "s/\[GITHUB_SHA\]/${GITHUB_SHA}/" bundle/.manifest
+      - name: Copy Rego files
+        run: |
+          shopt -s globstar
+          cp docker/policies/*.rego bundle/docker
+          cp kubernetes/lib/*.rego kubernetes/policies/general/*.rego bundle/kubernetes
+      - name: Compress
+        run: |
+          mkdir public
+          tar -C bundle -czvf public/bundle.tar.gz .
+      - name: Login to GitHub Packages Container registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ env.GH_USER }}
+          password: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
maybe use `oras login` instead? https://oras.land/implementors/#github-packages-container-registry-ghcr

Reply:
Internally `oras login` also does the same thing as `docker login`. `docker/login-action` is easier to read since we can pass a variable with a variable name and also logs out at the end of a job. So, I want to keep using `docker/login-action` for now. When some oras-specific processes will be necessary, I'll migrate to `oras login`.